### PR TITLE
Adding honest/dishonest personas

### PR DESCRIPTION
**Pull request by samNson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,93 @@
...
-Find the [Product Backlog](https://github.com/jorgecotillo/green_light_health/projects/1)
-under github projects.
-
-# Definition of Ready:
-Our D.O.R. requires that the team be able to determine "what needs to be done" and the "amount of work" required to complete a given story.
-The below checklist should be complete to ensure a story is Ready.
-- [ ] The story captures and describes a reasonable unit of work
-- [ ] The amount of work to complete the story has been estimated using the approved measurement (T-shirt Size)
-- [ ] The story is in line with the Product Vision
-- [ ] All blockers and dependencies for the needed implementation are resolved
```
Comment (nasadorian) [inline README.md:82]: @samNson With a whole file diff like this, it is hard to ascertain which lines have changed. Did you copy paste a new version of the file into your development branch?

File: README.md [hunk 2]
```diff
@@ -1,71 +1,93 @@
...
+Jason S  
+Bellevue WA  
+Age 20  
+Jason is a student at Bellevue College. Since the pandemic hit the US, his social life has shrunk and as a very
+extroverted person, he's been having a hard time with the isolation. Now that some public spaces and businesses are
+starting to open, he has been wanting to go out to eat and shop. However, most businesses are using the Green
+Light Health web app and requiring customers to declare their symptoms before they can be allowed to enter. Out
+of frustration, Jason has signed up for the app and he just wants to spend as less time as
+possible reading and filling out the health declaration forms. Because he doesn't want to be prevented entry, he just
+wants to fill the declaration always as having no symptoms, whether that's true or not.
```
Comment (nasadorian) [inline README.md:81]: What a bad guy!!

**Review by nasadorian — APPROVED** [review]
Great additions @samNson , just a comment about the diff for future PR readability

File: README.md (hunk 1 above)
Comment (samNson) [reply README.md:82]: @nasadorian no i merged master into my  branch before i added my personas. Maybe that caused the issue.

**Pull request merged**